api/v1alpha1/wavefront_types.go

```diff
@@ -73,9 +73,11 @@ type Metrics struct {
 	EnableDiscovery bool `json:"enableDiscovery,omitempty"`
 
 	// ClusterCollector is for resource configuration for the cluster collector.
+	// +kubebuilder:default:={resources: {requests: {cpu: "200m", memory: "10Mi", ephemeral-storage: "20Mi"}, limits: {cpu: "400m", memory: "512Mi", ephemeral-storage: "1Gi"}}}
```

Comment:
We chose the ephemeral-storage value to be twice as much as the memory following the examples mentioned in [this Openshift documentation](https://docs.openshift.com/dedicated/3/dev_guide/compute_resources.html). Is there a better way to arrive at the ephemeral-storage default values?

Reply:
We have decided to try with this and adjust if in the future something comes up.